dts/arm/microchip/mec172xnsz.dtsi

```diff
@@ -421,7 +421,7 @@
 			label = "TIMER_5";
 			max-value = <0xFFFFFFFF>;
 			prescaler = <0>;
-			status = "disabled";
+			status = "okay";
 		};
```

Comment:
status okay should be in the board level file?

Reply:
Agree. I'll move the code to board level dts file.